catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/SparkServlet.java

```diff
@@ -112,6 +120,8 @@ public void setFilterMappingPattern(String filterMappingPattern) {
 
     @Override
     public void init(ServletConfig config) throws ServletException {
+        registerProximityFactory();
```

Comment:
Any way we can move this to blueprint instead of putting it into the `SparkSevlet`?  There is a PR up on Spark to get the SparkServlet merged into their master so this file might be removed in the future.